_includes/tags.html

```diff
@@ -19,19 +19,19 @@
                        using the literal string '/blog/'. Path is used to
                        create the filtered URL: {{ path }}?filter_tags={{ tag }}.
 
-   display_header:     The optional boolean flag to display the tags_header element.
-                       Defaults to True. Adds class 'tags__hide-header' if falsy
-                       value passed (e.g., '', False, None).
+   hide_header:        The optional boolean flag to hide the tags_header element.
```

Comment:
We may want to change this to `hide_heading` at some point to better describe the type of content that is being hidden. Heading content is `<h1>`-`<h6>`, while a header is `<header>`.

Also, the defaults here and in https://github.com/cfpb/cfgov-refresh/pull/340/files#diff-ae11b16fd39f830f6f52d38e54892811R69 could be the same. I think defaulting to showing everything unless told to otherwise makes sense.

Neither of these need to hold up the PR I don't think.